scripts/common/web.ts

```diff
@@ -522,6 +523,33 @@ export enum GamemodeCategory {
 	DEFRAG = 8
 }
 
+export const GamemodeCategoryToGamemode = new Map<GamemodeCategory, Gamemode[]>([
```

Comment:
Do we have any documentation telling us to modify this when adding new game modes?

Reply:
I left comments in the C++ for adding gamemodes to update the website repo yeah. These changes to panorama are entirely generated by a bash script